commands.adoc

```diff
@@ -236,11 +244,11 @@ As per default, binaries are downloaded to
 To install client binary for Docker:
 
 ----------------------------------------------------------------------------------------------------
-$ vagrant service-manager install-cli docker
-# Binary already available at /home/johndoe/.vagrant.d/data/service-manager/bin/docker/1.9.1/docker
+$ vagrant service-manager install-cli docker --cli-version 1.9.1 --path /home/johndoe/docker
```

Comment:
What's about _/home/johndoe/bin/docker_ as an example dir. No one puts a binary directly into the home directory, right?